docs/development/setup.md

```diff
@@ -2,29 +2,29 @@
 
 ## Install the Java Development Kit
 
-The Deep Java Library (DJL)  project requires JDK 8 (or later). We recommend using JDK8, since there are some known issues with JDK 11+.
+The Deep Java Library (DJL)  project requires JDK 11 (or later). 
 
 Verify that Java is available in your $PATH environment variable by using the following commands. If you have multiple versions of Java installed,
 you can use the $JAVA_HOME environment variable to control which version of Java to use.
 
 For ubuntu:
 
 ```bash
-sudo apt-get install openjdk-8-jdk-headless
+sudo apt-get install openjdk-11-jdk
```

Comment:
Add headless to the end for size optimization

Reply:
We cannot use `-headless` to build DJL unless we update our CMake files. ALL JNI build requires `AWT` headers